doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -962,8 +946,9 @@ This operation supports two options:
 </td></tr>
 </table>
 
-Returns a list of entity types, either as JSON object (including the union set of attribute names along all the entities in each type) or
-a JSON array (just type name), depending on the `collapsed` option.
+Returns a JSON array of existing entity types. The elements of the array are either JSON objects (including
+the type name and the union set of attribute names along all the entities of such type) or
+strings (just type name), depending on the `collapsed` option.
 
```

Comment:
We need if we want to "select" between this two kind of responses:

```
[ "Room", "Car", ... ]
```

and

```
[ { type: "Car", attrs: [ ... ] }, { type: "Room", attrs: [ ... ] }, ... ]
```

After our talk on Friday I've gussed we want both, but maybe I missunderstood...